samples/prometheus-grafana/sample.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Capture and visualize metrics using Prometheus and Grafana
+component: Metrics
+---
+
+[Prometheus](https://prometheus.io) is a monitoring solution for storing time series data like metrics. [Grafana](https://grafana.com) allows to query and visualize the data stored in Prometheus (and other sources). This sample demonstrates how to capture NServiceBus metrics, storing these in Prometheus and visualizing these metrics using Grafana.
+
+
+
+![Grafana NServiceBus processing time](grafana-graph.png)
+
+
+
+This sample reports the following metrics to Prometheus:
+
+ * Fetched messages per second 
+ * Failed messages per second
+ * Successful messages per second
+ * Critical time in seconds
+ * Processing time seconds
+
+
+
+ The sample also shows data like `Average message processing time in milliseconds` and `Maximum critical time in milliseconds`. For a detailed explanation of these metrics refer to the [metrics captured section in the metrics documentation](/nservicebus/operations/metrics.md#metrics-captured).
+
+
+## Prerequisites
+
+To run this sample, download and run both Prometheus and Grafana. This sample uses Prometheus 1.7.1 and Grafana 4.4.2.
```

Comment:
I have no idea. I would say we can update from time to time and smoke test. Not sure how far we would go though because it might require running the sample again for 3 h to gather new graphs (if something in grafana changes visually that might affect the graphs)